### PR TITLE
dialogue when turns runs out NR S1 and S2

### LINKED ISSUE #5314: NR S2, lack of dialogue for out of time defeat
### Game and System Information
 - 1.15.7
 - Steam
 - Windows 10

### Describe the desired feature
NR S2 lacks any kind of dialogue to explain why you lose once turns run out. (The obvious choice would be food running out.)
I think it would be nice to add at least some explanation.

### PR DESCRIPTION
**Pull request by fraaans**
Fixes  #5314

### CODE REVIEW

**Comment by fraaans** [comment]
Fixes #5314

**Comment by knyghtmare** [comment]
S1: `Tallin knew what this sound means:`
Shouldn't it be "meant"?

S2: I am not sure if time passed by so fast in the caves. Would it more plausible if it was like Tallin and the rest being overwhelmed by a large number of powerful Skeletons such as Revenants, Deathblades or even Draug?

**Comment by fraaans** [comment]
@knyghtmare 
S1: for means/ meant i am not sure i think the 2 ways are correct but i preferred the one i wrote since the next line is in the present
S2: well the issue suggested food runs out so i added that they havent slept or eaten for a few days. I think if turns runs out it is safe to say some days could have gone by

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 1]
```diff
@@ -482,19 +482,19 @@
...
         [/message]
     [/event]
-    
+
     [event]
         name=time over
         [message]
             speaker= narrator
-            message= _ "Sounds of drums and horns echoes on the side of the mountains... Tallin knew what this sound means: it is too late! A large orc army is coming this way..."
+            message= _ "Sounds of drums and horns echoes on the side of the mountains... Tallin knew what this sound meant: it is too late! A large orc army is coming this way..."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:490]: I'd suggest avoiding too much extra punctuation unless there's a purpose to having it/you're trying to imply something with it. It makes the sentences hard to read. So, for example:

```suggestion
            message= _ "Sounds of drums and horns echo on the mountainsides. A large orc army draws near!"
```

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg [hunk 2]
```diff
@@ -482,19 +482,19 @@
...
         [/message]
-        
+
         [message]
             speaker=Tallin
             message= _ "If only we were faster escaping into the mines this wouldn't have happened..."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:495]: I think it's better to merge this line with the next one, it doesn't need to be 2 separate messages.

File: data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg [hunk 3]
```diff
@@ -1136,19 +1136,18 @@
...
         {CLEAR_VARIABLE found_Camerin,found_dwarves,trigger}
     [/event]
-    
+
     [event]
         name=time over
-        
         [message]
             speaker= narrator
             message= _ "Without eating or sleeping for days Tallin's men begin to fall one by one..."
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg:1144]:
```suggestion
            message= _ "Without eating or sleeping for days, Tallin's men begin to fall one by one."
```

File: data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg [hunk 4]
```diff
@@ -1136,19 +1136,18 @@
...
         [/message]
-        
+
         [message]
             speaker=Tallin
             message= _ "We were too slow to find the dwarves... I don't think many of us can still fight for very long!"
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg:1149]:
```suggestion
            message= _ "We were too slow in finding the dwarves. I don't think we can fight for much longer!"
```

**Review by nemaara — CHANGES_REQUESTED** [review]

File: data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg (hunk 2 above)
Comment (fraaans) [reply data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg:495]: thanks @nemaara

**Review by nemaara — APPROVED** [review]

**Comment by soliton-** [comment]
@nemaara When squashing commits it'd be good to also rewrite the commit message. The "Update fileX" entries are fairly useless.

**Pull request merged**